spec/ParseLiveQueryServer.spec.js

```diff
@@ -9,6 +9,16 @@ const queryHashValue = 'hash';
 const testUserId = 'userId';
 const testClassName = 'TestObject';
 
+const ASYNC_TEST_WAIT_TIME = 100;
+
+function resolveAfter(result, msTimeout) {
```

Comment:
Could we shorten this to
```
const timeout = t => new Promise(resolve => setTimeout(resolve, t || ASYNC_TEST_WAIT_TIME));

```
then just use
```
await timeout();

```

Reply:
We can, do you prefer `timeout` instead of `resolveAfter`?

Reply:
Also, is there a util file were this method should be moved? Or should we keep it file-specific?

Reply:
> We can, do you prefer timeout instead of resolveAfter?

Yes because this doesn't resolve anything anymore.

> Also, is there a util file were this method should be moved? Or should we keep it file-specific?

Good idea, we have a `helper.js` for global test declarations. I would however not move the default value, because we risk that someone changes that value in the future and suddenly tests become flaky or have a higher timeout than necessary.

Reply:
@mtrezza changes done